### PR TITLE
Fix unhandled promises in tests #171372354

### PR DESCRIPTION
**Pull request by LoganArnett**
This PR is for ticket [#171372354](https://www.pivotaltracker.com/story/show/171372354)

### CODE REVIEW

File: spec/javascript/apiService.test.js [hunk 1]
```diff
@@ -22,8 +22,11 @@ describe('apiService', () => {
       var expectedData = {'lease': lease}
 
       var result = apiService.createOrUpdateLease(lease, fakeAppId)
-
-      expect(result).resolves.toEqual(true)
+      // apiService.createOrUpdateLease has an async call
+      // that we then destructure and return instead of
+      // returning the result of the async call directly
+      // which returns undefined here instead of true
+      expect(result).resolves.toEqual(undefined)
```
Comment (akegan) [inline spec/javascript/apiService.test.js]: is there a way to change how we createOrUpdateLease so we can test it better? having it resolve to equal undefined doesn't seem like a great test 😕

Comment (LoganArnett) [reply spec/javascript/apiService.test.js]: Updated for another async method for testing this service

File: spec/javascript/apiService.test.js [hunk 2]
```diff
@@ -1,78 +1,97 @@
...
 
-      var result = apiService.createOrUpdateLease(lease, fakeAppId)
-      expect(result).resolves.toEqual(true)
-      expect(mockPostFn.mock.calls.length).toEqual(1)
-      expect(mockPostFn.mock.calls[0]).toEqual([`/applications/${fakeAppId}/leases`, expectedData])
+      var result = await apiService.createOrUpdateLease(lease, fakeAppId)
+      // apiService.createOrUpdateLease has an async call
+      // that we then destructure and return instead of
+      // returning the result of the async call directly
+      // which returns undefined here instead of true
```
Comment (akegan) [inline spec/javascript/apiService.test.js]: I think we can get rid of this comment now right?

**Review by akegan — APPROVED** [review]
Looks good. Two comments:
1. Can you update the PR title to follow [these standards](https://chris.beams.io/posts/git-commit/) for commit message best practices? e.g. something like "Fix unhandled promises in tests #171372354"
2. Added a comment that I think you can remove your comment about destructuring now.

File: spec/javascript/apiService.test.js (hunk 2 above)
Comment (LoganArnett) [reply spec/javascript/apiService.test.js]: yep, removed

**Pull request merged**